WordPress/gutenberg
# Add link to Post navigation block arrow icon

## What?
Fixes https://github.com/WordPress/gutenberg/issues/66040

## Why?
This PR adds link to Post navigation block previous and next arrow icons

## How?
Added the links to the arrows icons of Post navigation block

## Testing Instructions

1. Activate the Twenty-Twenty Five theme.
2. Insert the Post Navigation block pattern in the Site Editor.
3. Ensure the Next Post and Previous Post blocks are added.
4. Open a single post on the frontend.
5. Verify the navigation arrow icons are clickable and navigate to the same adjacent post.

## Screenshots or screencast 

**Before**
![post_no_link](https://github.com/user-attachments/assets/7c7162d9-d988-4c84-b8ef-f7227b062eaa)

**After**
![post_link](https://github.com/user-attachments/assets/0f17a78f-588b-4053-9797-59e5dbad9fdb)
